Makefile

```diff
@@ -0,0 +1,9 @@
+default: test
+
+bootstrap:
+	python setup.py develop
+
+setup:
```

Comment:
This make task is empty. Could it be removed or there a especial motive to have it here?